demo/znkExercise/demo.js

```diff
@@ -121,11 +122,6 @@
             };
 
             setExercise('practice', '165');
-        })
-        .run(function ($rootScope, $translate) {
```

Comment:
Oded remove it when he help me, probably by mistake (I'll ask him when he come)
I restore the code and check it and there was no error.